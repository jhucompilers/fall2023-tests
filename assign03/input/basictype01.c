int main(void) {
  char c1;
  signed char c2;
  unsigned char c3;

  c1 = -2;
  c2 = -3;
  c3 = 128;

  return 0;
}
