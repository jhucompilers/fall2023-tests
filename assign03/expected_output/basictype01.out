0|main|function|function () returning int
2|c1|variable|char
2|c2|variable|char
2|c3|variable|unsigned char
